packages/payment-processor/test/payment/single-request-proxy.test.ts

```
@@ -0,0 +1,271 @@
+  },
+  events: [],
+  expectedAmount: '100',
+  extensionsData: [],
+  meta: {
+    transactionManagerMeta: {},
+  },
+  pending: null,
+  requestId: 'abcd',
+  state: RequestLogicTypes.STATE.CREATED,
+  timestamp: 0,
+};
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)